packages/shell-api/src/database.ts

```diff
@@ -412,6 +412,14 @@ export default class Database extends ShellApiWithMongoClass {
     return await this._runAdminCommand(cmd, {});
   }
 
+  async aggregate(
+    singleStage: Document,
+    options?: Document
+  ): Promise<AggregationCursor>;
+  async aggregate(
+    pipeline: Document[],
+    options?: Document
+  ): Promise<AggregationCursor>;
```

Comment:
Totally optional, but as far as I can tell these overloads don't differ in return type or allowed parameter combinations, so you don't need to include them as explicit overloads:

```suggestion
```